client/components/Coverages/CoverageAddAdvancedModal.tsx

```diff
@@ -276,18 +278,26 @@ export class CoverageAddAdvancedModal extends React.Component<IProps, IState> {
                                                 </div>
 
                                                 <div className="grid__item grid__item--col4">
-                                                    <SelectUserInput
-                                                        field={'user'}
-                                                        value={coverage.user}
-                                                        onChange={(field, value) => this.onUserChange(coverage, value)}
-                                                        labelField="name"
-                                                        keyField="_id"
-                                                        users={coverage.filteredUsers}
-                                                        clearable={true}
-                                                        popupContainer={() => coverage.popupContainer}
-                                                        inline={true}
-                                                    />
-                                                    <div ref={(node) => coverage.popupContainer = node} />
+                                                    <div
+                                                        style={
+                                                            {
+                                                                margin: '0 0 1.8em 0',
+                                                                paddingBlockStart: '1.8rem',
+                                                                position: 'relative'}
+                                                        }
+                                                    >
+                                                        <SelectUser
+                                                            selectedUserId = {coverage.user?._id}
+                                                            onSelect={(user) => {
+                                                                this.onUserChange(coverage, user);
+                                                            }}
+                                                            autoFocus={false}
+                                                            horizontalSpacing={true}
+                                                            clearable={true}
+                                                            data-test-id= {'user'}
```

Comment:
Component doesn't have `data-test-id` prop defined in the interface. It already has "select-user-dropdown" data-test-id that you can use in tests when needed.